CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## [v1.2.1] - 2019-02-19
+
+- **CUMULUS-2078**
+  - Update release requirements to require cumulus-message-adapter >=1.2.0, <
```

Comment:
I think it would be nice if the `cumulus-message-adapter` text linked to that repo. But it's not strictly necessary